core/src/test/java/org/apache/calcite/test/SqlToRelConverterTest.java

```diff
@@ -962,6 +962,11 @@ class SqlToRelConverterTest extends SqlToRelTestBase {
     sql(sql).ok();
   }
 
+  @Test void testSelectCurrentTimestampIntervalWeek() {
+    final String sql = "select CURRENT_TIMESTAMP + INTERVAL '5 WEEKS' from emp";
+    sql(sql).ok();
+  }
+
```

Comment:
I'd prefer to have tests for both versions of the syntax, `interval 'X weeks'` and `interval 'X' weeks`